src/cmd/internal.go

```diff
@@ -0,0 +1,71 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/alecthomas/jsonschema"
+	"github.com/defenseunicorns/zarf/src/config"
+	"github.com/defenseunicorns/zarf/src/internal/git"
+	"github.com/defenseunicorns/zarf/src/internal/k8s"
+	"github.com/defenseunicorns/zarf/src/internal/message"
+	"github.com/defenseunicorns/zarf/src/types"
+	"github.com/spf13/cobra"
+	"github.com/spf13/cobra/doc"
+)
+
+var internalCmd = &cobra.Command{
+	Use:     "internal",
+	Aliases: []string{"dev", "int"},
```

Comment:
I suggest not including `int` since it could accidentally be trigger from a typo of `init` and lead to confusion. Up to your discretion though so feel free to resolve.